dev/yarn.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM yarnpkg/node-yarn:latest
+
+RUN mkdir /web
+WORKDIR /web
+COPY ./web /web
+RUN npm cache clean -f
+RUN npm install n -g -q
+RUN n stable -q
+RUN npm install -q
```

Comment:
This will install `node_modules` in the /web directory, but we seem to mount over that in docker-compose.